lib/fastlane/actions/update_icloud_container_identifiers.rb

```diff
@@ -0,0 +1,82 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      UPDATE_ICLOUD_CONTAINER_IDENTIFIERS = :UPDATE_ICLOUD_CONTAINER_IDENTIFIERS
+    end
+
+    class UpdateIcloudContainerIdentifiersAction < Action
+      require 'plist'
+
+      def self.run(params)
+        UI.message "Entitlements File: #{params[:entitlements_file]}"
+
+        entitlements_file = params[:entitlements_file]
+        raise "Could not find entitlements file at path '#{entitlements_file}'".red unless File.exist?(entitlements_file)
```

Comment:
I don't think this raise is necessary as this check exists in the verify block for the option.